voluncheer/opportunityboard/views/opportunityboard.py

```diff
@@ -98,11 +98,24 @@ def organization_view(request, userid):
     if user.is_organization:
         return redirect("home")
     volunteer = get_object_or_404(Volunteer, pk=user)
-    opportunity_lists = organization.opportunity_set.all()
+    recurring_posted_opportunity_lists = organization.opportunity_set.filter(
```

Comment:
I think you should keep the original name "opportunity list" so that it is still clear what this list is for.

Reply:
I agree!